content/docs/command-reference/init.md

```diff
@@ -102,18 +94,37 @@ project-A
 └── project-B
     ├── .dvc
     ├── data-B.dvc
-    │   ...
+    ...
 ```
 
-Nothing changes for the `project-B`. But for any DVC command being run in the
-`project-A` ignores the whole directory `project-B/`, meaning for example:
+Nothing changes for inner `project-B`. But any DVC command being run in outer
+`project-A` ignores the nested directory `project-B/`. For example, `dvc pull`
+(run in `project-A/`) wouldn't download data for the `data-B.dvc` file.
+
+### Nested DVC repositories
+
+Similarly, entire <abbr>DVC repositories</abbr> (each with it's own Git repo)
+can be nested. For example:
 
-```dvc
-$ cd project-A
-$ dvc pull
 ```
+project-A
+├── .git
+├── .dvc
+├── project-B
+│   ├── .git
+│   ├── .dvc
+│   ...
+├── project-C  # initialized with
+│   ├── .dvc   # --subdir or --no-scm
+...
+```
+
+> This is a questionable Git practice though, unless employing submodules.
 
-won't download or checkout data for the `data-B.dvc` file.
+In these cases all projects are also isolated from each other and commands run
+in one or the other only affects itself. Inner ones wouldn't search for
+data/pipelines above them anyway, and outer ones know to ignore sub-repos by
+default.
```

Comment:
Here's the main NEW text in my init change so far.

Reply:
But thinking about it maybe I can merge this with the previous section smoehow... ⏳

Reply:
> ones ignore sub-repos by default

I tested this with `dvc status` but I'm not sure it's 100% true with all commands... Need to test a few more things on this too ⏳

Reply:
> maybe I can merge this with the previous section

Done in 1f77e8fe.

Reply:
> not sure it's 100% true with all commands

Well, I'll just assume this is the case, at least it's the intention — otherwise it would be a bug anyway.